### ISSUE TITLE
Tailscale prevents the NAS device to sleep

### ISSUE THREAD

**Issue by Zorakie** [tailscale/tailscale#10133]
### What is the issue?

Hi I am fighting with a sleep mode after installation of Tailscale:

After installing Tailscale, the device is not able to sleep.

OOTB behavior of the device is, it falls asleep after a certain period of time, shuts down disks and everything, and only listens on Wake-on-LAN, so basically when you start a computer, that pings mapped drives, or you try to access the device from any already running computer, the NAS will wake up and spins disks.

After Tailscale installation, NAS will never sleep, it is constantly active, keeping disks alive.

For users that use the NAS as home backup storage (= accessing the disk effectively only several times a week at most), this is very inconvenient, as it does not result only in increased power consumption, but also unnecessary wearing off of the drives.

Tailscale should do what the native network does. Listen to Wake-on-LAN and only activate itself on an inbound connection (= if I start Tailscale on my phone, nothing should happen yet (if the phone does not ping the device on its own). If I try to access the device over Tailscale, Synology should wake up. Otherwise, when there is no request towards Synology over Tailscale, the device should be able to sleep.

Synology DS218Play
DSM version 7.2-64570 Update 3

### Steps to reproduce

Install and run Tailscale service.
Observe NAS LEDs after certain time of inactivity - never goes off, device never sleeps.
If you Stop/Uninstall the service, the device is able to sleep again after certain period of time.

### OS

Synology

### OS version

7.2-64570 Update 3

### Tailscale version

1.53.x

### Other software

OOTB setup, no extra services.

**Comment by DentonGentry** [comment]
We had two prior issues regarding buffering of telemetry, https://github.com/tailscale/tailscale/issues/7007 and https://github.com/tailscale/tailscale/issues/3551. Both of those have been resolved now, presumably this is something else.

**Comment by Zorakie** [comment]
@DentonGentry Thank you for your response. After reading those issues, it may be the same issue. Let's see. Can you please tell me the build number in which issues #7007 and #3551 were addressed, and I will try to upgrade? (My current build is 1.53.20). Then, I will let you know, if that solves the issue.

Also, If you need more logs from my device, feel free to request, only please tell me where to get them, as the Tailscale app page is rather...minimalistic.

Thx!

**Comment by DentonGentry** [comment]
Both were fixed long ago.
https://github.com/tailscale/tailscale/issues/3551 in 1.26
https://github.com/tailscale/tailscale/issues/7007 in 1.36

**Comment by Zorakie** [comment]
Ah, ok, so I am way beyond that as I see. Fine, then please let me know, if I can help with some logs/telementry to find out, what is going on :)

**Comment by Zorakie** [comment]
@DentonGentry I would like to update my statement:

I have caught 2x my NAS to actually go to sleep for a brief moment. The fact I haven't seen it before is, I have it in another room so basically any time I have been checking on it, it was fully awake.

Anyway: It seems like the NAS is actually trying to go to sleep, and from time to time it actually goes to sleep for a moment (all LEDs except of one go off). But that lasts for like 30 seconds, before it wakes up again.

During that last time I caught it,, I am 100% sure there was no other active device in the network, both local and Tailscale, ALL computers in the network were OFFLINE, and my phone was connected to data, not wifi and tailscale was off on the phone.

So basically the only 3 active network devices at that time were: NAS, Router, Switch.

That being said, isn't possible, that Tailscale servers would execute any kind of heartbeat probe regularly, that would trigger this waking up, as the NAS thinks it is some actual incoming request?

**Comment by andrewelisity** [comment]
I face the same issue. After installing Tailscale 1.38.4-700038004 on DSM 7.2.1-69057 Update 4 HDDs never spin down to hibernate

**Comment by Zorakie** [comment]
Btw the issue still persists on 1.62.1 :(

**Comment by 0x20F** [comment]
Same issue here with tailscale 1.58.2-1, and DSM 7.2.1-69057 Update 5.

**Comment by 0x20F** [comment]
I updated to tailscale 1.69.66-700069066 (unstable) and that seems to have fixed the issue for me at least.
